notebooks/00_quick_start/tfidf_covid.ipynb

```diff
@@ -0,0 +1,1235 @@
+{
```

Comment:
<p>I'm going to check this internally to make sure this is the right way of managing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1088/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>